lib/clamp-link.in

```diff
@@ -152,10 +153,8 @@ do
     fi
 
     if [[ "$ARG" =~ ^-L\/[^[:space:]]+$ ]]; then
-      # this is an absolute path
-      LIB_SEARCH_PATHS+=( ${ARG:2} )
-    else
-      LIB_SEARCH_PATHS+=( $PWD"/"${ARG:2} )
+      REAL_PATH=$(realpath ${ARG:2})
```

Comment:
realpath is installed on Ubuntu by default. To prevent yet another external dependency, could we try get the canonical path via:

```
REAL_PATH=$(readlink -f ${ARG:2})
```